app/notifications/consumer.js

```diff
@@ -0,0 +1,44 @@
+const ampq = require('ampqlib')
+const logger = require('../utils/logger')
+const { EXCHANGE_NAME, NOTIFICATIONS_RK } = require('exchangeConfig')
+const transporter = require('../utils/mailer')
+const emailComposer = require('../utils/composeEmail')
+
+class Consumer {
+  static async proceedNotification () {
+    const connection = await ampq.connect('amqp://localhost')
+    const channel = await connection.createChannel()
+    try {
+      await channel.checkExchange(EXCHANGE_NAME)
+    } catch (err) {
+      logger.error(err)
+      process.exit(1)
+    }
+    try {
+      await channel.checkQueue(NOTIFICATIONS_RK)
+    } catch (err) {
+      logger.error(err)
+      process.exit(1)
+    }
+    try {
+      await channel.consume(NOTIFICATIONS_RK, async (msg) => {
+        const letter = emailComposer(msg)
+        try {
+          await transporter.sendMail(letter, (err, info) => {
```

Comment:
sendMail doesn't return the promise and try catch block couldn't get an error in async code.
`await` doesn't help.